docs - improves hub docs (#3282)
dlt-hub/dlt: docs/website/docs/hub/getting-started/installation.md

## Quickstart

To install the `dlt[workspace]` package, create a new [Python virtual environment](#setting-up-your-environment) and run:
```sh
uv pip install "dlt[workspace]"
```
This will install `dlt` with several additional dependencies you'll need for local development: `arrow`, `marimo`, `mcp`, and a few others.

> If you need to install `uv` (a modern package manager), [please refer to the next section](#configuration-of-the-python-environment)

Reviewer: ```suggestion
If you need to install `uv` (a modern package manager), [please refer to the next section](#configuration-of-the-python-environment).
```

> ### Enable free tier features

Reviewer: ```suggestion
### Enable dltHub Free tier features
```

:::info
New [free tier features](../intro.md#tiers--licensing) like profiles are hidden behind a feature flag,
which means you need to manually enable them before use.

> To activate these features, create the `.dlt/.workspace` file in your project directory; this tells `dlt` to switch from the classic project mode to the new Workspace mode.

Reviewer: ```suggestion
To activate these features, create an empty `.dlt/.workspace` file in your project directory; this tells `dlt` to switch from the classic project mode to the Workspace mode.
```

```sh
uv venv
```
This will create a virtual environment in the `.venv` folder using the default system Python version.

```sh
uv venv --python 3.13
```
This will use `Python 3.13` for your virtual environment.

Activate the virtual environment using the instructions displayed by `uv`, i.e.:

```sh
source .venv/bin/activate
```

## Licensing 

To access dltHub’s paid features, such as Iceberg support or Python-based transformations, you need a dltHub Software License.

1. Contact us if you want to purchase a license or get a trial license with unlimited use.
2. Issue a [limited trial license](#self-licensing) yourself.

#### Install your license

If you've received your license from us, you can install it in one of two ways:

In the `secrets.toml` file:
```toml
license = "your-dlthub-license-key"
```

As an environment variable:
```sh
export DLT_LICENSE_KEY="your-dlthub-license-key"
```

#### Features requiring a license:

- [@dlt.hub.transformation](../features/transformations/index.md) - a powerful Python decorator to build transformation pipelines and notebooks

> - [dbt transformations](../features/transformations/dbt-transformations.md): a staging layer for data transformations, combining a local cache with schema enforcement, debugging tools, and integration with existing data workflows.

Reviewer: ```suggestion
- [dbt transformations](../features/transformations/dbt-transformations.md) - a staging layer for data transformations, combining a local cache with schema enforcement, debugging tools, and integration with existing data workflows.
```

- [Iceberg support](../ecosystem/iceberg.md)

For more information about the feature scopes, see [Scopes](#scopes).

> Please also review our End User License Agreement [(EULA)](../EULA.md)

Reviewer: ```suggestion
Please, also review our [End User License Agreement](../EULA.md)
```

```sh
dlt license issue <scope>
```

for example:
```sh
dlt license issue dlthub.transformation
```
This will do the following:
* Issue a new license (or merge with existing scopes) for the [transformations](../features/transformations/index.md) feature.

> * Print your license key in the CLI output

Reviewer: ```suggestion
* Print your license key in the CLI output.
```
